stories/demos/components/Toolbar/Toolbar.tsx

```diff
@@ -2,32 +2,19 @@ import React, { PropsWithChildren } from 'react';
 import './rdg-toolbar.less';
 
 interface Props {
-  onAddRow?: (arg: { newRowIndex: number }) => void;
-  onToggleFilter?: () => void;
-  enableFilter?: boolean;
+  onAddRow: (arg: { newRowIndex: number }) => void;
```

Comment:
I wonder if we need to keep the `Toolbar` component. We can probably add the button directly on the parent component